dmcontent/content_loader.py

```diff
@@ -396,8 +396,11 @@ def get_data(self, form_data):
 
     def _get_single_question_data(self, form_data):
         if self.id not in form_data and self.type != 'boolean_list':
-            if self.get('assuranceApproach') and '{}--assurance'.format(self.id) in form_data:
-                return {self.id: {'assurance': form_data.get('{}--assurance'.format(self.id))}}
+            if self.get('assuranceApproach'):
+                if '{}--assurance'.format(self.id) in form_data:
+                    return {self.id: {'assurance': form_data.get('{}--assurance'.format(self.id))}}
+                else:
+                    return {self.id: {'value': [] if self.type in ['list', 'checkboxes'] else None}}
```

Comment:
Do we need to add 'value' key here? Looking at the unpacking method, it should handle the missing key fine, and the API will return `answer_required` either way since `value` is a required key.